x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.js

```diff
@@ -230,11 +230,18 @@ export function geoShapeToGeometry(value, accumulator) {
   let geoJson;
   if (typeof value === 'string') {
     geoJson = convertWKTStringToGeojson(value);
+    accumulator.push(geoJson);
```

Comment:
Since `accumulator.push(geoJson)` was moved into each if/else block, it no longer makes sense to have `geoJson` outside of the scope of its respected block. Lets remove geoJson declaration on line 230 and then declare geoJson in the if block in line 232 and else block in line 242